src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -171,6 +172,7 @@ class SidebarLinks extends React.Component {
                         option => option.reportID === activeReportID
                     ))}
                     onSelectRow={(option) => {
+                        markStart(CONST.TIMING.SWITCH_REPORT, option);
```

Comment:
Should maybe happen here instead just to keep the marks together:

https://github.com/Expensify/App/blob/bd863d61b3e0161dcce0e0085f02a10923bea876/src/pages/home/sidebar/SidebarScreen.js#L86-L88